tests/org.eclipse.xtend.core.tests/src/org/eclipse/xtend/core/tests/validation/ValidationBug427432Test.xtend

```diff
@@ -31,7 +31,7 @@ class ValidationBug427432Test extends AbstractXtendTestCase {
 		'''
 			class Foo {
 			
-				val x = 1
+				static val x = 1
```

Comment:
I don't share the opinion that this is ok, since the second case in the test model is dead code but no longer flagged as such.